README.md

```diff
@@ -8,23 +8,21 @@ Welcome to [LMHT].  This is an open source code repository to help Spanish speak
 
 ## Introduction
 
-The [LMHT page] provides a list of current HTML tags.  Select the tag you are looking for or begin typing it in the search box.  Once selected, a separate window will pop up containing the english base of the tag, the spanish equivalent, a definition in both english and spanish as well as audio recordings of each of these items.  Furthermore, there are helpful hints on how to remember what the tag does as well as examples of the code being used.
+The [LMHT site] provides a list of current HTML tags. Select the tag you are looking for or begin typing it in the search box. Once selected, an embedded window will appear, containing the English base of the tag, the Spanish equivalent, a definition in both English and Spanish, as well as audio recordings of each of these items.  Furthermore, there are helpful hints on how to remember what the tag does as well as examples of the code being used.
 
 [LMHT page]:  http://lmht.github.io/
```

Comment:
This needs to be changed from [LMHT page] to [LMHT site] based on the changes submitted.